server/schedulers/utils.go

```diff
@@ -137,8 +138,27 @@ func (p *balancePlan) getTolerantResource() int64 {
 	return regionSize
 }
 
+<<<<<<< HEAD
```

Comment:
need resolve conflict.

Reply:
it is related to #3745, I also cherry pick it.